IronFrame.Test/Utilities/JobObjectTests.cs

```diff
@@ -223,6 +224,62 @@ public void WhenMemoryIsNotLimited_GetMemoryPeakReturnsZero()
             }
         }
 
+        public class JobCpuLimits : IDisposable
+        {
+            JobObject jobObject;
+            JobObject jobObject2;
+
+
+            public JobCpuLimits()
+            {
+                jobObject = new JobObject();
+                jobObject2 = new JobObject();
+            }
+
+            public void Dispose()
+            {
+                jobObject.Dispose();
+                jobObject2.Dispose();
+            }
+
+            [Fact]
+            public void CanLimitCpu()
+            {
+                jobObject.SetJobCpuLimit(1);
+                jobObject2.SetJobCpuLimit(9);
+
+                var thread1 = new Thread(() =>
```

Comment:
We initially tried this and discovered that `ExecuteInJob` blocks until the process exits, that's why we moved them into threads.